plugins/gatsby-source-content-chunks/gatsby-node.js

```diff
@@ -0,0 +1,48 @@
+const getFileRelativePath = (absolutePath) =>
```

Comment:
It looks like you're no longer using this plugin, so feel free to delete the source code for it.